vowpalwabbit/core/src/reductions/cb/cb_explore_adf_squarecb.cc

```diff
@@ -225,6 +226,9 @@ void cb_explore_adf_squarecb::predict(multi_learner& base, VW::multi_ex& example
       total_weight += pa;
     }
     preds[a_min].score = 1.f - total_weight;
+
+    exploration::enforce_minimum_probability(_epsilon, true, begin_scores(preds), end_scores(preds));
+    // TODO: Do we need to sort here?
```

Comment:
is this comment still valid?

Reply:
I think it can be removed, just need to check